vortex-api/src/main/java/mil/army/usace/hec/vortex/io/DssDataWriter.java

```diff
@@ -768,176 +765,157 @@ private List<TimeSeriesContainer> getTscs() {
-            if (isRegular) {
-                setRegularTscTimes(thirdQuartileTsc, diff, startTimes, type);
-            } else {
-                setIrregularTscTimes(thirdQuartileTsc, endTimes);
-            }
+                TimeSeriesContainer tscGreaterThanZero = initTsc("-PCT>0", pctCellsGreaterThanZero, type);
+                tscGreaterThanZero.setUnits("%");
 
-            if(options.get("75th Percentile").equalsIgnoreCase("true") && !Double.isNaN(thirdQuartileTsc.getValue(0))) {
-                tscs.add(thirdQuartileTsc);
+                tscs.add(tscGreaterThanZero);
             }
 
-            double[] pctCellsGreaterThanZero = zonalStatistics.stream()
-                    .mapToDouble(ZonalStatistics::getPctCellsGreaterThanZero)
-                    .toArray();
+            if (options.getOrDefault("Pct>1Q", "").equalsIgnoreCase("true")) {
+                double[] pctCellsGreaterThan1Q = zonalStatistics.stream()
+                        .mapToDouble(ZonalStatistics::getPctCellsGreaterThanFirstQuartile)
+                        .toArray();
 
-            TimeSeriesContainer pctCellsGreaterThanZeroTsc = createTsc("-PCTGREATERZERO", pctCellsGreaterThanZero, type);
-            if (isRegular) {
-                setRegularTscTimes(pctCellsGreaterThanZeroTsc, diff, startTimes, type);
-            } else {
-                setIrregularTscTimes(pctCellsGreaterThanZeroTsc, endTimes);
-            }
+                TimeSeriesContainer tscGreaterThan1Q = initTsc("-PCT>Q1", pctCellsGreaterThan1Q, type);
+                tscGreaterThan1Q.setUnits("%");
 
-            if(options.get("Percentage of Cells > 0").equalsIgnoreCase("true") && !Double.isNaN(pctCellsGreaterThanZeroTsc.getValue(0))) {
-                tscs.add(pctCellsGreaterThanZeroTsc);
+                tscs.add(tscGreaterThan1Q);
             }
 
-            double[] pctCellsGreaterThanFirstQuartile = zonalStatistics.stream()
-                    .mapToDouble(ZonalStatistics::getPctCellsGreaterThanFirstQuartile)
-                    .toArray();
+            // Set times for each TimeSeriesContainer
+            for (TimeSeriesContainer tsc : tscs) {
+                if (isRegular) {
+                    setRegularTscTimes(tsc, diff, startTimes.get(0), type);
+                } else {
+                    setIrregularTscTimes(tsc, endTimes);
+                }
+            }
 
-            TimeSeriesContainer pctCellsGreaterThanFirstQuartileTsc = createTsc("-PCTGREATERQ1", pctCellsGreaterThanFirstQuartile, type);
-            if (isRegular) {
-                setRegularTscTimes(pctCellsGreaterThanFirstQuartileTsc, diff, startTimes, type);
-            } else {
-                setIrregularTscTimes(pctCellsGreaterThanFirstQuartileTsc, endTimes);
+            // Add an accumulation cumulative precipitation TimeSeriesContainer
+            if (options.getOrDefault("isAccumulate", "false").equalsIgnoreCase("true")) {
```

Comment:
Is checking if "isAccumulate" true necessary? It looks like it is always true based on GridToPointWizard.